examples/bluegreen/bluegreen-controller.yaml

```diff
@@ -12,6 +12,8 @@ spec:
     kind: BlueGreenDeployment
     shortNames:
     - bgd
+  subresources:
```

Comment:
Is this necessary? It looks like BGD test passes even without status subresource.

Reply:
Yes. The test only passes because we failed to test that status works. If you have time, you could add some checks in `test.sh` to verify that the BGD status is updated correctly. Right now, it's only looking at the underlying RS statuses.